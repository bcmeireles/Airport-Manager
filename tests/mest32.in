a LIS Portugal Lisboa
a POR Portugal Lisboa
v TT6 LIS POR 14-02-2022 14:00 2:00 200
v TT7 LIS POR 15-02-2022 14:00 2:00 120
v TT6 LIS POR 17-02-2022 14:00 2:00 150

r TT6 17-02-2022 123456789A 147
r TT6 17-02-2022 123456789B 1
r TT6 17-02-2022 123456789C 1
r TT6 14-02-2022 123456789E 180
r TT6 14-02-2022 123456789D 20


t 15-02-2022
r TT7 15-02-2022 123456789E 1
e 123456789E
r TT7 15-02-2022 123456789E 1

r TT6 14-02-2022
r TT6 14-02-2022 123456789Z 1


e TT6
r TT7 15-02-2022 123456789A 1
r TT7 15-02-2022 123456789B 1
r TT7 15-02-2022 123456789C 1
r TT7 15-02-2022 123456789D 1



q