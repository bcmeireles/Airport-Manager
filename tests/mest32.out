airport LIS
airport POR
15-02-2022
123456789E: flight reservation already used
invalid date
invalid date
